src/screens/ContactScreen/ContactScreen.tsx

```diff
@@ -1,32 +1,93 @@
 import React from 'react';
-import { Text } from 'react-native';
-import { SpacerInline } from '@elements';
-import InfoScreen from '../InfoScreen';
-import styles from './ContactScreen.styles';
+import {
+	ScrollView,
+	Text,
+	TouchableHighlight,
+	View,
+} from 'react-native';
+import { Linking } from 'expo';
+import {
+	ContentHeader,
+	NavBar,
+	SpacerInline,
+	Icon,
+} from '@elements';
+import { IconName } from '@elements/Icon';
+import styles, { ListItem } from './ContactScreen.styles';
+
+const contactList: Array<{
+	title: string;
+	message: string;
+	link: string;
+	iconName: IconName;
+}> = [
+	{
+		title: 'website',
+		message: 'www.bananaapp.org',
+		link: 'https://www.bananaapp.org',
+		iconName: 'website',
+	},
+	{
+		title: 'email',
+		message: 'info@bananapp.org',
+		// TODO: Supposedly this link works on real devices, not simulators. Verify.
```

Comment:
The email feature works on iPhone 6 with Apple's default email app.
However I uninstalled my Apple's default email app on my phone and instead use the Gmail app so this feature failed for me. It's likely that most people will not have deleted their default email app, so it seems like this is probably not a big issue?

Reply:
I'll write the TODOs with a link to your comment! Thank you for testing this well :)

Reply:
these were comments from the OG implementation. but, I'll add a TODO with a link to your comment :) Thanks for the testing!

Reply:
No prob! 👍